lib/gui/models/selection-state.js

```diff
@@ -25,7 +25,8 @@ const angular = require('angular');
 const Store = require('./store');
 const MODULE_NAME = 'Etcher.Models.SelectionState';
 const SelectionStateModel = angular.module(MODULE_NAME, [
-  require('./drives')
+  require('./drives'),
+  require('./drive-constraints')
```

Comment:
I just realized we're not using the drive constraints model anywhere here, so I guess it should be removed.